src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/forgotpassword/DetachedForgotPasswordPage.java

```diff
@@ -0,0 +1,53 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth.forgotpassword;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.DetachedWindow;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.FormPage;
+
+public class DetachedForgotPasswordPage extends DetachedWindow implements ForgotPasswordPage {
+
+  private AttachedForgotPasswordPage forgotPasswordPage;
+
+  private final static String TITLE = "Forgot password";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)